commcare_export/misc.py

```diff
@@ -41,7 +41,7 @@ def unwrap_val(val):
         if len(val) == 1:
             val = val[0]
         else:
-            val = map(unwrap_val, val)
+            val = list(map(unwrap_val, val))
```

Comment:
Nitpick: would be better as a list comprehension `[unwrap_val(v) for v in val]` as to only iterate over the items once (in Python 2)